datafusion/expr/src/test/function_stub.rs

```diff
@@ -326,7 +326,7 @@ impl Default for Min {
 impl Min {
     pub fn new() -> Self {
         Self {
-            aliases: vec!["min".to_string()],
```

Comment:
We don't need uppercase alias.

To use uppercase alias the query is something like `select "MAX"(a) from t;`, which is uncommon.